### PR TITLE
(QA-2355) add classifiers

### PR DESCRIPTION
**Pull request by jpinsonault**
Based off #4, it should be merged before this one. Only look at the last commit.

Adds classifier meta-data to setup.py

### CODE REVIEW

File: setup.py [hunk 1]
```diff
@@ -3,18 +3,28 @@
...
 from vmpooler_client.version import version
 
+classifiers = [
+    'Development Status :: 5 - Production/Stable',
+    'Intended Audience :: End Users/Desktop',
+    'Topic :: Utilities',
+    'License :: OSI Approved :: Apache Software License',
+    'Programming Language :: Python :: 2',
+    'Programming Language :: Python :: 2.7',
+]
```
Comment (cowofevil) [inline setup.py]: Does not follow PEP8 block assignment formatting.

File: setup.py [hunk 2]
```diff
@@ -3,18 +3,28 @@
...
+from setuptools import setup
 from vmpooler_client.version import version
 
+classifiers = [
+    'Development Status :: 5 - Production/Stable',
+    'Intended Audience :: End Users/Desktop',
+    'Topic :: Utilities',
+    'License :: OSI Approved :: Apache Software License',
+    'Programming Language :: Python :: 2',
+    'Programming Language :: Python :: 2.7',
```
Comment (cowofevil) [inline setup.py]: I think maybe we could claim Python 3.x support.

File: setup.py [hunk 3]
```diff
@@ -3,18 +3,28 @@
 #===================================================================================================
 # Imports
 #===================================================================================================
-from distutils.core import setup
+from setuptools import setup
 from vmpooler_client.version import version
 
+classifiers = [
```
Comment (cowofevil) [inline setup.py]: Being picky here:

I think that the classifiers should be under a comment header "Globals". The ```classifiers``` variable name should be all caps to indicate that it is a global.

File: setup.py (hunk 2 above)
Comment (jpinsonault) [inline setup.py]: In python 3.4 at least it throws this error:

```
Traceback (most recent call last):
  File "./vmpooler_client_app.py", line 16, in <module>
    from vmpooler_client.command_parser import CommandParser, valid_lifetime
  File "/Users/joepinsonault/puppetlabs/vmpooler-client/vmpooler_client/command_parser.py", line 14, in <module>
    from util import MAX_LIFETIME
ImportError: No module named 'util'
```

File: setup.py (hunk 1 above)
Comment (jpinsonault) [inline setup.py]: I couldn't find what pep8 block assignment formatting is with a quick google, but the pep8 linter doesn't complain. Is that the hanging comma or the hanging bracket?

File: setup.py (hunk 2 above)
Comment (cowofevil) [inline setup.py]: Alrighty then, we'll need to make sure we can get this to work on 3.x in the near future.

File: setup.py (hunk 1 above)
Comment (cowofevil) [inline setup.py]: The hanging bracket. We already established the formatting style in our other Python modules.

**Comment by jpinsonault** [comment]
Rebased and addressed comments

**Comment by jpinsonault** [comment]
Rebased

**Pull request merged**